github action failed when install dependencies

Install dependencies 炸了

```
2025-10-31T17:35:56.1840356Z ##[group]Run uv sync
2025-10-31T17:35:56.1840595Z [36;1muv sync[0m
2025-10-31T17:35:56.1867995Z shell: /usr/bin/bash -e {0}
2025-10-31T17:35:56.1868251Z env:
2025-10-31T17:35:56.1868485Z   UV_CACHE_DIR: /home/runner/work/_temp/setup-uv-cache
2025-10-31T17:35:56.1868839Z   pythonLocation: /opt/hostedtoolcache/Python/3.14.0/x64
2025-10-31T17:35:56.1869251Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.14.0/x64/lib/pkgconfig
2025-10-31T17:35:56.1869626Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.14.0/x64
2025-10-31T17:35:56.1869956Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.14.0/x64
2025-10-31T17:35:56.1870294Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.14.0/x64
2025-10-31T17:35:56.1870628Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.14.0/x64/lib
2025-10-31T17:35:56.1870911Z ##[endgroup]
2025-10-31T17:35:56.2969295Z Using CPython 3.14.0 interpreter at: /opt/hostedtoolcache/Python/3.14.0/x64/bin/python3
2025-10-31T17:35:56.2971305Z Creating virtual environment at: .venv
2025-10-31T17:35:56.3025261Z Resolved 104 packages in 0.66ms
2025-10-31T17:35:56.3405560Z Downloading ruff (10.8MiB)
2025-10-31T17:35:56.3417090Z Downloading playwright (43.7MiB)
2025-10-31T17:35:56.3432331Z Downloading pygments (1.2MiB)
2025-10-31T17:35:56.3452980Z Downloading fonttools (1.1MiB)
2025-10-31T17:35:56.3459952Z Downloading pillow (6.3MiB)
2025-10-31T17:35:56.3469560Z Downloading opencv-python (63.9MiB)
2025-10-31T17:35:56.6926520Z    Building msgpack==1.1.1
2025-10-31T17:35:56.7721518Z    Building maxminddb==2.7.0
2025-10-31T17:35:56.7847801Z    Building kiwisolver==1.4.8
2025-10-31T17:35:56.9037704Z    Building pydantic-core==2.33.2
2025-10-31T17:35:57.0169591Z  Downloading pygments
2025-10-31T17:35:57.0293929Z  Downloading fonttools
2025-10-31T17:35:57.2343034Z    Building pyyaml==6.0.2
2025-10-31T17:35:57.5153622Z    Building lxml==6.0.0
2025-10-31T17:35:58.2267730Z  Downloading pillow
2025-10-31T17:35:58.6796401Z    Building aiohttp==3.12.14
2025-10-31T17:35:59.5007030Z  Downloading ruff
2025-10-31T17:35:59.9956347Z    Building contourpy==1.3.2
2025-10-31T17:36:01.6286416Z    Building orjson==3.11.0
2025-10-31T17:36:02.7789743Z   × Failed to build `lxml==6.0.0`
2025-10-31T17:36:02.7791596Z   ├─▶ The build backend returned an error
2025-10-31T17:36:02.7800487Z   ╰─▶ Call to `setuptools.build_meta:__legacy__.build_wheel` failed (exit
2025-10-31T17:36:02.7801951Z       status: 1)
2025-10-31T17:36:02.7802801Z 
2025-10-31T17:36:02.7803684Z       [stdout]
2025-10-31T17:36:02.7804612Z       Building lxml version 6.0.0.
2025-10-31T17:36:02.7805893Z       Building without Cython.
2025-10-31T17:36:02.7807118Z       Error: Please make sure the libxml2 and libxslt development packages
2025-10-31T17:36:02.7808337Z       are installed.
2025-10-31T17:36:02.7809131Z 
2025-10-31T17:36:02.7810280Z       hint: This usually indicates a problem with the package or the build
2025-10-31T17:36:02.7812794Z       environment.
2025-10-31T17:36:02.7845919Z   help: `lxml` (v6.0.0) was included because `epic-awesome-gamer` (v0.4.1)
2025-10-31T17:36:02.7859076Z         depends on `hcaptcha-challenger[camoufox]` (v0.18.10) which depends on
2025-10-31T17:36:02.7861457Z         `camoufox` (v0.4.11) which depends on `lxml`
2025-10-31T17:36:02.8151552Z ##[error]Process completed with exit code 1.

```

--- a/.github/workflows/epic-gamer.yml
+++ b/.github/workflows/epic-gamer.yml
@@ -66,16 +66,15 @@ jobs:
         with:
           python-version-file: "./epic-gamer-src/pyproject.toml"
 
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y xvfb libxml2-dev libxslt-dev
+
       # Install dependencies
       - name: Install dependencies
         working-directory: ./epic-gamer-src
         run: uv sync
-
-      # Install system dependencies for browser automation
-      - name: Install system dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y xvfb
 
       # Install Playwright browsers with retry logic
       - name: Install Playwright browsers

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -7,7 +7,7 @@ dependencies = [
     "celery[redis]>=5.4.0",
     "hcaptcha-challenger[camoufox]>=0.18.10",
 ]
-requires-python = ">=3.12"
+requires-python = ">=3.12,<=3.13"
 authors = [
     { name = "QIN2DIM", email = "yaoqinse@gmail.com" }
 ]

